packages/cli/src/index.ts

```diff
@@ -197,8 +299,7 @@ cli.on('command:*', async () => {
 // We only keep the original code environment.
 preventExperimentalWarning();
```

Comment:
Can this method help me delete it? We don’t need this now.

Reply:
Of course, I'll delete it later